POC/script/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[project]
+name = "script"
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+license = "Apache-2.0"
+
+[dependencies]
+fuel-core = { version = "0.9", default-features = false }
```

Comment:
Is `fuel-core` used?

Reply:
`fuel-core` isn't used, you're right. Fixed [739649f](https://github.com/FuelLabs/bridge/pull/6/commits/739649f7131d3c5d097c71496b3cf4b78e490c3c)